src/controllers/dishController.ts

```diff
@@ -14,17 +11,27 @@ export const getAllDishes = async (
   }
 };
 
-export const createDish = async (
-  req: express.Request,
-  res: express.Response
-) => {
+export const getDishByIngredient = async (req: express.Request, res: express.Response) => {
+  try {
+    const { ingredient } = req.params;
+    const dishes = await DishModel.find();
+    const dishesWithIngredient = dishes.filter((dish) => {
+      const ingredients = dish.ingredients.split(",").map((dish) => dish.trim().toLowerCase());
+      return ingredients.includes(ingredient.toLowerCase());
```

Comment:
u'll need to lower case the ingredient that u're getting from the params if the user is typing with big letters

Reply:
fixed